### PR TITLE
[CPDNPQ-1752] Styling fixes after frontend update

### PR DESCRIPTION
**Pull request by TBenedyk**
### Context

Ticket: https://dfedigital.atlassian.net/browse/CPDNPQ-1752

- Tags should be sentence case.
- Questions should be h2.
- Questions should have medium text if there is an XL title above them.

### Changes proposed in this pull request

Question text configuration was already set but was being overridden in `_radio_button_group.html.erb` because of the ordering of arguments. They also needed changing from h1 to h2.

**Before:**
Screenshot 2024-04-22 at 11 36 00

**After:**
Screenshot 2024-04-22 at 11 36 13

**Before:**
Screenshot 2024-04-22 at 11 36 50

**After:**
Screenshot 2024-04-22 at 11 36 38

**Before:**
Screenshot 2024-04-22 at 11 37 23

**After:**
Screenshot 2024-04-22 at 11 37 34

**Before:**
Screenshot 2024-04-22 at 11 45 27

**After:**
Screenshot 2024-04-22 at 11 45 06

**Before:**
Screenshot 2024-04-22 at 12 35 01

**After:**
Screenshot 2024-04-22 at 12 05 08

### CODE REVIEW

**Comment by peteryates** [comment]
Coincidentally I just raised [another PR](https://github.com/DFE-Digital/npq-registration/pull/1316#pullrequestreview-2014509265) that did some tag case fixing 😅

File: app/views/accounts/_course_details.html.erb [hunk 1]
```diff
@@ -89,10 +89,10 @@
           </dt>
           <dd class="govuk-summary-list__value">
               <% if application.participant_outcome_state.capitalize == "Passed" %>
-                <strong class="govuk-tag govuk-tag--green">PASSED</strong>
+                <strong class="govuk-tag govuk-tag--green">Passed</strong>
```
Comment (peteryates) [inline app/views/accounts/_course_details.html.erb:92]: We can use govuk-components here, it's already installed and set up:

```suggestion
                <%= govuk_tag(text: "Passed", colour: "green") %>
```

Comment (TBenedyk) [reply app/views/accounts/_course_details.html.erb:92]: OK thanks I'll update them

**Review by peteryates — COMMENTED** [review]
This looks great. I left a comment about using govuk_components.

The other thing is on the final screenshot I think having both the heading and subheading (legend) is unnecessary

image

Maybe one for @Dom-Collins to cast his eye over (can be addressed as a follow up)

File: app/views/accounts/_course_details.html.erb (hunk 1 above)
Comment (peteryates) [reply app/views/accounts/_course_details.html.erb:92]: Cheers, [here's a link to the gem docs](https://govuk-components.netlify.app/components/tag/).

**Review by slawosz — APPROVED** [review]

**Review by peteryates — APPROVED** [review]
Looks great. @Dom-Collins - mind taking a look at the page I referenced above and seeing if it's worth tweaking in a follow up PR?

**Comment by dalcJ** [comment]
@TBenedyk All looks good, thank you :)

**Pull request merged**